c_binding/libata.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2016 Red Hat, Inc.
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
```

Comment:
@tasleson @joehandzik Thanks. I will change license to "LGPL v2.1 or later" including the NVMe library.